src/style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  margin: 0;
+  display: block;
+}
+
+.card {
+  width: 198px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+}
+
+.card__image {
+  margin: 32px 19px 40px;
+
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  margin: 0 16px 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  margin: 0 0 18px 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__stars-and-reviews-container {
+  margin: 0 16px 22px 18px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-image: url("./images/star.svg");
+}
+
+.stars__star:not(:last-of-type) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url("./images/star-active.svg");
+}
+
+.card__reviews {
+  text-align: right;
```

Comment:
It's unnecessary to set that value because the element is already in the right corner due to the parent justify-content property.